build/build.py

```diff
@@ -81,6 +81,12 @@ def __init__(self):
             help='Override default build output directory'
         )
 
+        '''SDK only'''
+        self.parser.add_argument(
+            '--disable-exceptions', dest='disable_exceptions', action='store_true',
+            help='Builds libdeliveryoptimization without compiling exception throwing code into the binary'
+        )
+
```

Comment:
I went with this convention because this allows for opting out of exceptions, and it doesn't break the current interface.

By default, both the throwing and non-throwing public apis will be available